python/tests/res/enkf/data/test_gen_kw_config.py

```diff
@@ -34,3 +34,23 @@ def test_gen_kw_config(self):
             conf = GenKwConfig("KEY", template_file, parameter_file)
             conf = GenKwConfig("KEY", template_file, parameter_file_comments)
             self.assertEqual(len(conf), 3)
+    
+    
+    def test_gen_kw_config_get_priors(self):
+        with TestAreaContext("gen_kw_config"):
```

Comment:
Also, the `tmpdir` construct from `tests.utils` is preferred unless `TestAreaContext` is truly necessary.